Microsoft.Research/Contracts/WindowsBase/System.Windows.Rect.cs

```diff
@@ -16,6 +16,19 @@
 using System.ComponentModel;
 using System.Diagnostics.Contracts;
 
+
+// 2015-03-36: tom-englert
+// Temporarily disabled checks for Contract.Requires(!this.IsEmpty);
```

Comment:
:exclamation: I'm not sold on this particular change. This pull request is primarily intended to add missing contracts and remove incorrect contracts, but this change is a "hack" which removes a *correct* contract. I prefer to address this (all instances where this contract was changed) by the following in order of preference:

1. Address the limitation which results in the described warning (not likely to happen).
2. Find a place where we can put `Contract.Assume` to help out the checker.
3. "Just deal with the warning", or similar
4. Move the changes to their own pull request, which can be reverted should the situation change in the future.